utilities/consume/dataflow_fhirstore_bq_consume/README.md

```diff
@@ -0,0 +1,40 @@
+# DataFlow Streaming Pipeline to read from Healthcare API FHIR Store via BQ and write to Google Cloud Storage
+
+DataFlow streaming pipeline consumes Pub/Sub FHIR notification(Full Resource or NameOnly) from Google Cloud Healthcare API FHIR Store, query the data from BigQuery and outout daat to Google Cloud Storage. Users can add other sinks as per their requirements
+
+
+# Architecture
+ 
+ ![Log output image](./images/dataflow_fhirstore_bq_consume.png)
+
+ 1. Configure Healthcare FHIR store to send Pub/Sub notifications. FHIR notifications are generated on each action on FHIR store (Create, Update, Delete)
+ 2.  Configure Healthcare FHIR store to write to BigQuery. Each FHIR resource will have there own table eg: all CarePlan fhir resources will be ingested in CarePlan BigQuery table. A BigQuery row is appended for each action on FHIR store.
+ 3. Dataflow reads from Pub/Sub subscription and extracts resource id.
```

Comment:
Do we need to include what is resource id for end customers  like with examples (e.g Patient/adhsff-efbhf-0dfgg)?

Reply:
will add, thanks